src/components/sujoEmbedded/sujoEmbedded.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";

```

Comment:
Coding style thing: We haven't used trailing semi-colons so those could be removed to keep it consistent.

Reply:
Updated. Also for other lines. This file was copy from current site.